frontend/src/scenes/session-recordings/player/list/consoleLogsUtils.tsx

```diff
@@ -55,63 +55,41 @@ export function parseEntry(entry?: string): ParsedEntry {
-            splitString.forEach((splitPart, index) => {
-                if (urlIndices.includes(index)) {
-                    const url = splitPart.replace(/^\(/, '').replace(/\)$/, '') // remove flanking parentheses
-                    let shortenedURL
-                    const splitTrace = url.split(':')
-                    const lineNumbers = splitTrace.slice(-2).join(':').split(/\s+/)[0]
-                    const baseURL = splitTrace.slice(0, -2).join(':')
-                    if (splitTrace.length >= 4) {
-                        // Case with URL and line number
-                        try {
-                            const fileNameFromURL = new URL(baseURL).pathname.split('/').slice(-1)[0]
-                            shortenedURL = `${fileNameFromURL}:${lineNumbers}`
-                        } catch (e) {
-                            // If we can't parse the URL, fall back to this line number
-                            shortenedURL = `:${lineNumbers}`
-                        }
-                    } else if (splitTrace.length === 3) {
-                        // Case with line number only
-                        shortenedURL = `:${lineNumbers}`
-                    }
-                    const link = (
-                        <a href={baseURL} target="_blank">
-                            {shortenedURL}
-                        </a>
-                    )
-                    element = (
-                        <>
-                            {element} {link}
-                        </>
-                    )
-                    if (!traceUrl) {
-                        traceUrl = link
-                    }
-                } else {
-                    element = (
-                        <>
-                            {element} {splitPart}
-                        </>
-                    )
+            const url = splitEntry.replace(/^\(/, '').replace(/\)$/, '') // remove flanking parentheses
+            let shortenedURL
+            const splitTrace = url.split(':')
+            const lineNumbers = splitTrace.slice(-2).join(':').split(/\s+/)[0]
+            const baseURL = splitTrace.slice(0, -2).join(':')
+            if (splitTrace.length >= 4) {
+                // Case with URL and line number
+                try {
+                    const fileNameFromURL = new URL(baseURL).pathname.split('/').slice(-1)[0]
+                    shortenedURL = `${fileNameFromURL}:${lineNumbers}`
+                } catch (e) {
+                    // If we can't parse the URL, fall back to this line number
+                    shortenedURL = `:${lineNumbers}`
                 }
-            })
-            return element
+            } else if (splitTrace.length === 3) {
+                // Case with line number only
+                shortenedURL = `:${lineNumbers}`
+            }
+            const link = (
+                <a href={baseURL} target="_blank">
+                    {shortenedURL}
+                </a>
```

Comment:
(Needs import so just a suggestion)
```suggestion
                <Link to={baseURL} target="_blank">
                    {shortenedURL}
                </Link>
```